ui/pages/home/beta/beta-home-footer.component.js

```diff
@@ -1,21 +1,18 @@
 import React from 'react';
-import { useI18nContext } from '../../hooks/useI18nContext';
+import { SUPPORT_REQUEST_LINK } from '../../../helpers/constants/common';
+import { useI18nContext } from '../../../hooks/useI18nContext';
 
 const BetaHomeFooter = () => {
   const t = useI18nContext();
 
   return (
     <>
-      <a
-        href="https://metamask.zendesk.com/hc/en-us/requests/new"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
+      <a href={SUPPORT_REQUEST_LINK} target="_blank" rel="noopener noreferrer">
         {t('needHelpSubmitTicket')}
       </a>{' '}
       |{' '}
       <a
-        href="https://community.metamask.io/c/metamask-beta/30"
+        href="https://community.metamask.io/c/metamask-beta"
```

Comment:
Interesting. Not sure what the trailing `/30` was - maybe the category number? Seems to work without it.

Reply:
I was playing around on the community site and the number seemed to vary with the amount of things displayed on the page? In any event, it does seem inessential to get the user to where we want to go. The category sub-pages appear to appear under their alphabetical names in the URL.